SportsHub/.idea/.idea.SportsHub/.idea/encodings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
check, please, why are these files here

Reply:
I staged them by mistake when i pushed from my Rider . But they should probably be ignored from what i have seen they are in the gitignore

Reply:
but they also still here